src/components/TheFooter.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <v-app>
+    <v-footer padless>
+      <v-card
+        tile
+        flat
+        width="100%"
+        color="grey"
+        dark
+        height="100px"
+        class="d-flex align-center justify-center"
+        style="position: absolute; bottom: 0; left: 0; right: 0"
```

Comment:
When you have a lot a props and attributes in a component you can sort them in alphabetical order just to improve readability. Also, the boolean props (`dark`, `flat` and `title`) don't have the `=` sign, so they can be put together as well.

```suggestion
        class="d-flex align-center justify-center"
        color="grey"
        height="100px"
        style="position: absolute; bottom: 0; left: 0; right: 0"
        width="100%"
        dark
        flat
        tile
```